samples/MQTTSN/README.md

```diff
@@ -0,0 +1,114 @@
+## Clone this repository
+```
+git clone https://github.com/t-meitan-msft/azure-iot-udp-samples.git
+```
+
+## Build the Paho MQTT-SN Client Sample
+
+In a Linux environment, run:
+
+```
+cd azure-iot-udp-samples/samples/MQTTSN/cmake

```

Comment:
this directory does not exists "azure-iot-udp-samples/samples/MQTTSN/cmake"

cd azure-iot-udp-samples/samples/MQTTSN
mkdir cmake
cd cmake
cmake ..
cmake --build . (or make)